src/Miunie.ConsoleApp/Resources/Lang/PhrasesEn.json

```diff
@@ -23,7 +23,8 @@
     {
         "Key": "CANNOT_SELF_REP",
         "Pool": [
-            "Sorry {0} you can't give rep to yourself!"
+            "Sorry {0} you cannot influence your own reputation! Try influencing other people instead. :smirk:",
```

Comment:
Sorry {0}, you cannot influence your own reputation!

Just adding a comma would make it neater.